config/application.example.yml

```diff
@@ -1,11 +1,23 @@
 default: &default
+  DECIDIM_APPLICATION_NAME: Decidim Clean APP
+#  DECIDIM_MAILER_SENDER: ??????
+  DECIDIM_AVAILABLE_LOCALES: "ca,en,es"
+  DECIDIM_DEFAULT_LOCALE: "ca"
+  DECIDIM_CORS_ENABLED: "true"
+  DECIDIM_TRACK_NEWSLETTER_LINKS: auto
+  DECIDIM_THROTTLING_MAX_REQUESTS: "250"
+  # DECIDIM_THROTTLING_PERIOD in minutes
+  DECIDIM_THROTTLING_PERIOD: "1"
+  DECIDIM_FOLLOW_HTTP_X_FORWARDED_HOST: "true"
+
   # Database configuration
   DB_PORT: "5432"
-  DB_POOL: <%= ENV.fetch("RAILS_MAX_THREADS") { "5" } %>
+  DB_POOL: "<%= ENV.fetch("RAILS_MAX_THREADS") { "5" } %>"
   DB_HOST: <%= ENV.fetch("DATABASE_HOST") { "localhost" } %>
-  DB_DATABASE: "database name"
-  DB_USERNAME: "database username"
-  DB_PASSWORD: "database password"
+  DB_DATABASE: "clean_app_dev"
+  DB_TEST_DATABASE: "clean_app_test"
+  DB_USERNAME: "db_username"
+  DB_PASSWORD: "db_username_password"
```

Comment:
can you review examples names? there are related to clean app but maybe is better use "Pinto" names

Reply:
It's ok, this way we don't have conflicts when merging the clean-app